### DISCUSSION TITLE
Station construction

### DISCUSSION THREAD

**Discussion by Gamma-Andromedae** [Opinionated-Blueprints/10-Books-Full-of-Rails#119] [Q&A] [answered]
Hello,

I'm trying to build an odd configuration of a station in which I have 3 requester stations sharing a stacker of size 4 between them. I'm wondering if I should set the train limit on the requesters to 5 on each or if that would cause the combination to try calling 15 trains at maximum throughput. Any tips on how to make this work or alternative ideas would be appreciated.
Thanks!

**Comment by brandon-gradient** [comment, marked as answer]
The combined total of stations + stackers should equal train limit of all 3 stations combined. So all 3 stations combined shouldn't exceed 7 when combining their individual train limits in your case. As an example if you set train limit to 3 for each, then they each could have up to 3 trains waiting, if your throughput is low enough, and the 6 extra trains would fill up and "overflow" the stacker
